ansible/setup.yml

```diff
@@ -111,7 +111,7 @@
         - ansible
         - boto
         - boto3
-        - datadog==0.28.0
+        - datadog
```

Comment:
Have you verified the new version works ? I bet it's frozen for a reason

Reply:
what was reason, what should be checked?

Reply:
i missed that change, i think there were some breaking changes that is why it was locked, should be checked if it works -> send properly data to datadog, may be investigate the changelog of the agent

Reply:
probably leave that for a separate PR

Reply:
@dincho @venimus  i have locked version back :D changing is not worth testing ;]